src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,58 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
 
+        // Test MovieService
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
```

Comment:
just note for the future, it is better to initialize our services outside of the method